sscanss/app/window/presenter.py

```diff
@@ -266,15 +266,16 @@ def importSample(self):
         insert_command = InsertSampleFromFile(filename, self, insert_option)
         self.view.undo_stack.push(insert_command)
 
-    def importTomography(self, filepath, sizes_and_centres):
+    def importTomography(self, filepath, pixel_sizes, pixel_centres):
```

Comment:
please replace with `def importTomography(self, filepath, pixel_sizes=None, pixel_centres=None):` also update docstring to `Optional[List[float, float, float]]`